src/status_im2/contexts/syncing/sheets/sync_device_notice/views.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.contexts.syncing.sheets.sync-device-notice.views
+  (:require
+   [quo.react-native :as rn]
+   [status-im2.contexts.syncing.sheets.sync-device-notice.styles :as styles]
+   [status-im2.contexts.syncing.sheets.enter-password.views :as enter-password]
+   [utils.re-frame :as re-frame]
+   [quo2.core :as quo]
+   [i18n.i18n :as i18n]))
+
+(defn views []
```

Comment:
sheet

Reply:
> sheet

@flexsurfer :  Do you mean I should rename views function to sheets or the file name to sheets?

Reply:
function

Reply:
btw it should view.cljs, not views.cljs